### PR TITLE
Integrating with common

### PR DESCRIPTION
**Pull request by ustulation**
This change is [Reviewable](https://reviewable.io/reviews/maidsafe/safe_vault/377)

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @maqi

(maidsafe_highfive has picked a reviewer for you, use r? to override)

File: Cargo.toml [hunk 1]
```diff
@@ -11,20 +11,23 @@ version = "0.4.0"
...
 maidsafe_utilities = "~0.4.0"
-mpid_messaging = "~0.2.0"
 routing = "~0.11.1"
 rustc-serialize = "~0.3.18"
+safe_network_common = "~0.0.1"
 sodiumoxide = "~0.0.9"
 time = "~0.1.34"
 xor_name = "~0.1.0"
 
+[dependencies.clippy]
```
Comment (Fraser999) [inline Cargo.toml]: Can you revert this style please?  All other repos are using the single line style.  Also I think we're up to 0.0.54 (although that might have changed by the time I write this :)  ).

File: src/main.rs [hunk 2]
```diff
@@ -61,6 +60,7 @@ extern crate rustc_serialize;
 extern crate sodiumoxide;
 extern crate time;
 extern crate xor_name;
+extern crate safe_network_common;
```
Comment (maqi) [inline src/main.rs]: i think it is preferred that crates listed in an alphabet order?

**Comment by maqi** [comment]
Reviewed 1 of 12 files at r1, 8 of 11 files at r2.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/maidsafe/safe_vault/377)*

**Pull request merged**